skops/card/_model_card.py

```diff
@@ -380,8 +380,17 @@ class Card:
         ``Path``/``str`` of the model or the actual model instance that will be
         documented. If a ``Path`` or ``str`` is provided, model will be loaded.
 
-    model_diagram: bool, default=True
-        Set to True if model diagram should be plotted in the card.
+    model_diagram: bool or "auto" or str, default="auto"
+        If using the skops template, setting this to ``True`` or ``"auto"`` will
+        add the model diagram, as generated by sckit-learn, to the default
+        section. Passing a string to ``model_diagram`` will instead use that
```

Comment:
We should add the section name here.

```suggestion
        section, i.e. {SECTION_NAME}. Passing a string to ``model_diagram`` will instead use that
```

Reply:
Done